target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -42,6 +42,10 @@ case "$FIRMWARE" in
 		;;
 	netgear,rbr40|\
 	netgear,rbs40|\
+	linksys,whw03)
+		caldata_extract_mmc "0:ART" 0x9000 0x2f20
+		ath10k_patch_mac $(macaddr_setbit_la $(macaddr_add "$(cat /sys/class/net/eth0/address)" 4))
+		;;
 	netgear,rbr50|\
```

Comment:
Seems you're separating the Netgear Orbi family here. Intentionally?

Reply:
that was an auto-merge byproduct of the cherry-pick - no conflict, but not what was intended, I have fixed this, appreciate the review :)

Reply:
This also changes the functionality for the `netgear,rbs40` and `netgear,rbr40`, please add the new entry before the netgear devices.